### PR TITLE
Einstein Probe mission page

### PR DESCRIPTION
**Pull request by jracusin**
This is a start to the EP mission page.  Please provide edits and additional details to include.

### CODE REVIEW

File: app/routes/_gcn.missions.einsteinprobe/logo.png
Comment (lpsinger) [inline app/routes/_gcn.missions.einsteinprobe/logo.png:1]: Is there an SVG version of this image?

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — CHANGES_REQUESTED** [review]
Please fix Prettier issues.

**Comment by lpsinger** [comment]
@modot, would you please check this?

File: app/routes/_gcn.missions.einsteinprobe/logo.png
Comment (jracusin) [reply app/routes/_gcn.missions.einsteinprobe/logo.png:1]: I have only found a png.

**Comment by jracusin** [comment]
> Please fix Prettier issues.

done

**Comment by modot** [comment]
> @modot, would you please check this?

I'll check it in the morning.

**Comment by modot** [comment]
Hi @jracusin, I just sent an email containing the .svg logos and a patch for the mission page.

**Comment by jracusin** [comment]
> Hi @jracusin, I just sent an email containing the .svg logos and a patch for the mission page.

I see the email chain, but not the email with the file.  Maybe the attachments were too large that they got blocked.  Could you add them via GitHub?

File: app/routes/_gcn.missions.einsteinprobe/route.mdx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+</div>
+
+**Yearly Trigger Rates:**
+
+<div className="overflow-table">
+
+| Instrument | Type                                                   | Rates |
+| ---------- | ------------------------------------------------------ | ----- |
+| WXT        | GRBs, other transients, and non-astrophysical triggers | ??    |
```
Comment (jracusin) [inline app/routes/_gcn.missions.einsteinprobe/route.mdx:50]: @modot Could you provide information for us to include on rates?

**Comment by lpsinger** [comment]
Please rebase.

**Comment by jracusin** [comment]
Rebase had too many conflicts due to change to file structures, so moved this to new PR #2172

**Pull request closed without merging**